pkg/rke2/np.go

```diff
@@ -120,13 +122,44 @@ var networkIngressPolicy = v1.NetworkPolicy{
 					{
 						Protocol: &tcp,
 						Port: &intstr.IntOrString{
-							IntVal: int32(80),
+							StrVal: "http",
```

Comment:
note: targeting by name is more correct, as the numeric ports can be set by chart values.

https://github.com/rancher/rke2-charts/blob/3a5a54512273934ab59711b44e969c9ae84c94fe/charts/rke2-ingress-nginx/rke2-ingress-nginx/4.8.200/values.yaml#L36-L37